ckan/logic/action/create.py

```diff
@@ -220,7 +220,7 @@ def package_create(context, data_dict):
             {'package': data})
 
     # Create activity
-    if not pkg.private:
+    if not pkg.private or config.get('ckan.record_private_activity', False):
```

Comment:
You need an asbool here for 2.9 because a string will be returned (typed configuration options code is only in master)

Reply:
@wardi added. (I did not have an issue with this locally on ckan2.9 but I am doing some per-proccessing of the ini file in my docker setup because of dynamic values so that might be why)